src/motion/bfgs_optimizer.F

```diff
@@ -260,13 +273,34 @@ RECURSIVE SUBROUTINE geoopt_bfgs(force_env, gopt_param, globenv, geo_section, go
       ALLOCATE (dr(ndf))
       dr(:) = 0.0_dp
 
+      ! find space_group
+      CALL section_vals_val_get(geo_section, "KEEP_SPACE_GROUP", l_val=keep_sg)
```

Comment:
I think, it would be helpful to generally rename `keep_sg` into `keep_space_group`.

Reply:
The renaming is done.